src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/DeserializedAggregateValidator.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    /// <summary>
+    /// <see cref="IAggregateValidator"/> implementation returned by deserialization.
+    /// </summary>
+    public class DeserializedAggregateValidator : IAggregateValidator
+    {
+        [JsonProperty("name")]
+        public string Name { get; }
+
+        public Task<AggregateValidationResult> Validate(ValidationContext data)
+        {
+            throw new NotImplementedException();
```

Comment:
What's the purpose of this class, if it has no implementation? Same question for DeserializedValidator...